Formula/a/azion-cli.rb

```diff
@@ -0,0 +1,34 @@
+class AzionCli < Formula
```

Comment:
if you are not sure about this name, maybe you should ask first?

https://github.com/Homebrew/homebrew-core/pull/152353